elcid/assets/js/elcid/services/records/diagnosis.js

```diff
@@ -7,7 +7,7 @@ angular.module('opal.records')
         return function(record){
             if(!record.id){
                 if(!record.date_of_diagnosis){
-                    record.date_of_diagnosis = moment()
+                    record.date_of_diagnosis = moment().toDate();
```

Comment:
this should be a moment right? We use moment's in the records and dates in the forms?